src/iop/colorzones.c

```diff
@@ -984,7 +984,13 @@ static void _draw_background(cairo_t *cr, dt_iop_colorzones_params_t *p, dt_iop_
                              const int select_by_picker, const int width, const int height,
                              const float *picked_color)
 {
-  const float normalize_C = (128.f * sqrtf(2.f));
+  float bg_sat_factor = dt_conf_get_float("plugins/darkroom/colorzones/bg_sat_factor");
+  if (bg_sat_factor == 0) {
```

Comment:
This should not be needed. Please add an entry into `data/darktableconfig.xml.in` (no prefs / section attribute).

type should be float, default should be 0.5 and min set to 0.1 and max to 1.

See `plugins/darkroom/masks/heightview` for example.

When this is done you just have to take the value (dt_conf_get_float) and not worry about the initial value and even if 0 since the value will be checked to be in the defined bound each time dt starts.